### PR TITLE
Use personal access token for GitHub rebase command

### PR DESCRIPTION
**Pull request by msasikanth**
Reference: https://github.com/cirrus-actions/rebase/pull/70

### CODE REVIEW

**Comment by msasikanth** [comment]
Currently, I have created a PAT and added it to secrets. Feel free to replace that if needed.

File: .github/workflows/ci_check_pr_comment.yml [hunk 1]
```diff
@@ -32,15 +32,17 @@ jobs:
...
       - uses: actions/checkout@v2
         with:
+          token: ${{ secrets.PAT }}
+          persist-credentials: false
           fetch-depth: 0
       - name: Rebase
         uses: cirrus-actions/rebase@1.4
         env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_ACCESS_TOKEN }}
+          GITHUB_TOKEN: ${{ secrets.PAT }}
```
Comment (vinaysshenoy) [inline .github/workflows/ci_check_pr_comment.yml:45]: We already have a personal access token secret `GITHUB_ACCESS_TOKEN`, which we added for the code formatting checks (iirc). Is that not usable for this check?

Comment (msasikanth) [reply .github/workflows/ci_check_pr_comment.yml:45]: Oh, if it's already a personal access token then it should work. I thought it was a different one. Let me replace it with that.

**Review by vinaysshenoy — APPROVED** [review]

File: .github/workflows/ci_check_pr_comment.yml (hunk 1 above)
Comment (vinaysshenoy) [reply .github/workflows/ci_check_pr_comment.yml]: Also, please avoid using abbreviations like `PAT` and use the full name like `PERSONAL_ACCESS_TOKEN`. Someone looking at the list of secrets a month from now will have no idea what `PAT` actually is.

**Pull request merged**